alerts/ui/src/AppContent.jsx

```diff
@@ -48,17 +43,9 @@ const AppContent = () => {
   const { setActiveSelectedTab } = useGlobalsActions()
   const activeSelectedTab = useGlobalsActiveSelectedTab()
 
-  useEffect(() => {
-    if (!authError) return
-    addMessage({
-      variant: "error",
-      text: parseError(authError),
-    })
-  }, [authError])
-
   useEffect(() => {
     // since the API call is done in a web worker and not logging aware, we need to show the error just in case the user is logged in
-    if (!alertsError || !loggedIn) return
+    if (!alertsError) return
 
     // if user uses firefox warn to activate `allow_client_cert`. Should be enough to do it just here since the API call is done in a web worker and nothing else will be loaded until the alerts are loaded
     const isFirefox = navigator.userAgent.toLowerCase().includes("firefox")
```

Comment:
I just realized that this is also very specific to our auth setup. We should move this out of Supernova as well.